src/main/java/com/github/javafaker/Address.java

```diff
@@ -1,7 +1,10 @@
 package com.github.javafaker;
 
+import com.github.javafaker.service.RandomService;
+
 public class Address {
     private final Faker faker;
+    private final RandomService random = new RandomService();

```

Comment:
Can you pass the randomservice in instead of creating a new one.. You should be able to use the one inside the faker constructor

Reply:
Fixed.